openshift/topics/introduction.adoc

```diff
@@ -1,35 +1,66 @@
-== Introduction
+== Introduction to Red Hat for Single Sign-On for OpenShift Image
 
 === What Is {project_name}?
+[role="_abstract"]
 {project_name} is an integrated sign-on solution available as a Red Hat JBoss Middleware for OpenShift containerized image. The {project_openshift_product_name} image provides an authentication server for users to centrally log in, log out, register, and manage user accounts for web applications, mobile applications, and RESTful web services.
 
-{openshift_name} is only available on the following platforms: *{openshift_image_platforms}*. For other available platforms, see link:{openshift_link_other}[{openshift_name_other}].
+{openshift_name} is available on *{openshift_image_platforms}*. For other available platforms, see link:{openshift_link_other}[{openshift_name_other}].
+
+=== Comparison: {project_openshift_product_name} Image versus Red Hat Single Sign-On
+The {project_openshift_product_name} image version number {project_version} is based on {project_name} {project_version}. There are some differences in functionality between the {project_openshift_product_name} image and {project_name}:
```

Comment:
Replace `There are some differences...` with `There are some important differences...` or with `There are some difference in functionality ... and {project_name}, that should be considered:` ?